app/views/directives/osc-persistent-volume-claim.html

```diff
@@ -180,6 +187,11 @@
                Can't be greater than {{limits.max | usageWithUnits : 'storage'}}.
              </span>
            </div>
+           <div ng-if="persistentVolumeClaimForm.capacity.$error.outOfStorage" class="has-error">
```

Comment:
the variable names are a little confusing given what they are actually doing... "outOfClaims" is really am I out of either claims or storage, and "outOfStorage" is really will my request exceed my storage quota.  I think at least changing "outOfStorage" to "willExceedStorage" would help